verl/trainer/fsdp_sft_trainer.py

```diff
@@ -165,6 +193,14 @@ def _build_model_optimizer(self):
         trust_remote_code = self.config.model.trust_remote_code
         # load config first
         config = AutoConfig.from_pretrained(local_model_path, trust_remote_code=trust_remote_code)
+        if self.use_remove_padding:
+            assert self.config.ulysses_sequence_parallel_size > 1, "Remove padding is only supported with sequence parallel"
```

Comment:
I assume it should be the opposite? Sequence parallel is only support when remove_padding is enabled?